task-2/src/components/SortingFilter/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { useDispatch } from 'react-redux';
+
+import {
+	fetchMovies,
+	moviesSortedByTitle,
+	moviesSortedByDate,
+	moviesSortedByRating,
+} from '../../store/thunk';
+
+export const SortingFilter = () => {
+	const dispatch = useDispatch();
+
+	const sortingHandler = (sortedValue) => {
+		switch (sortedValue) {
+			case 'title':
+				dispatch(moviesSortedByTitle());
+				break;
+			case 'date':
+				dispatch(moviesSortedByDate());
+				break;
+			case 'rating':
+				dispatch(moviesSortedByRating());
+				break;
+			case 'all':
+				dispatch(fetchMovies());
+		}
+	};
+	return (
+		<aside>
+			<select
+				onChange={(e) => {
```

Comment:
You don't need to have an arrow function in JSX that just passes "e.target.value" to another arrow function.
Move the "e.target.value" part to the "sortingHandler".

Reply:
fixed.